### ISSUE TITLE
Improve the Update and Get syntax

### ISSUE THREAD

**Issue by dumconstantin** [code11/engine#53]
Provide some shorthands and capabilities for `Update` and `Get` in order to make more powerful constructs:

```
foo = Update.foo.value
foo.set('123')
foo('123')
```
So you can call `foo` without `set` to achieve the same thing.

Also add array methods on `Update` to simplify working with array:
```
foo = Update.foo.value
foo.pop()
foo.push('123')
foo.unshift()
....
```

`Get` on the other hand can have an extended syntax like `Update`:
```
foo = Get.foo.value
foo() // this is now

// but you could also have this:
foo.contains('123')
foo.isValid()
foo.length()
```

**Comment by dumconstantin** [comment]
toggle is another candidate for Update as described in #5

**Comment by dumconstantin** [comment]
resolved by #93
